com.microsoft.mrtk.uxcore/Interop/UGUIInputAdapterDraggable.cs

```diff
@@ -35,8 +35,11 @@ public virtual void OnDrag(PointerEventData pointerEventData)
             // and we don't want to duplicate them.
             if (IsXRUIEvent(pointerEventData)) { return; }
 
-            // We only adapt drags for selectable interactables.
-            if (!(ThisInteractable is IXRSelectInteractable selectInteractable)) { return; }
+            // We only adapt drags for selectable interactables,
+            // and if we have a valid proxy interactor.
+            if (ThisInteractable is IXRSelectInteractable selectInteractable &&
```

Comment:
Is this missing a leading `!` modifier? The CI pipeline is also returning errors.

Reply:
Ope